client/query/query.go

```diff
@@ -99,12 +118,19 @@ func GetPaymentChannelInfo(id types.Destination, store store.Store, vm *payments
 			Balance: getPaymentChannelBalance(defund.VFixed.Participants, []outcome.SingleAssetExit{defund.FinalOutcome}),
 		}, nil
 	}
-
 	// Otherwise we can just check the store
 	c, channelFound := store.GetChannelById(id)
 
 	if channelFound {
 		status := getStatusFromChannel(c)
```

Comment:
Previously we were checking if `c.PostFundComplete()` to determine if the channel status was "Ready" or "Proposed". However since the changes for ADR 0009 intermediaries exit the protocol early before receiving all the postfund signatures.

This means we can rely on `c.PostFundComplete()` to determine the status of a virtual channel.